bindings/binding_core_node/src/minify.rs

```diff
@@ -89,18 +91,15 @@ fn minify(code: Buffer, opts: Buffer, signal: Option<AbortSignal>) -> AsyncTask<
 pub fn minify_sync(code: Buffer, opts: Buffer) -> napi::Result<TransformOutput> {
     crate::util::init_default_trace_subscriber();
     let code: MinifyTarget = get_deserialized(code)?;
-    let opts = get_deserialized(opts)?;
+    let opts_str = String::from_utf8_lossy(opts.as_ref()).to_string();
+    let opts: JsMinifyOptions = deserialize_json(&opts_str)?;
```

Comment:
Please revert this. `minifySync` need to do all operations on the JS thread, so we deserialize json from the JS thread, but `minify()` is about delegating the work to the non-main thread.